plexapi/media.py

```diff
@@ -433,10 +433,34 @@ def setSelected(self):
         """
         return self._parent().setSelectedAudioStream(self)
 
+    def levels(self, subSample=128):
+        """ Load time series loudness levels.
+
+            Attributes:
+                subSample: (int): the number of loudness segments to load        
+        """
+
+        key = f'/library/streams/{self.id}/levels'
+        params = {'subsample': subSample}
+
+        levels = self.fetchItems(ekey=key, cls=Level, params=params)
+        return levels
+
     @deprecated('Use "setSelected" instead.')
     def setDefault(self):
         return self.setSelected()
 
+@utils.registerPlexObject
+class Level(PlexObject):
+    """ Represents an instance of loudness for an audio stream.
+
+        Attributes:
+            loudness (float): loudness level in db
+    """
+    def _loadData(self, data):  
+        """ Load attribute values from Plex XML response. """
+        self.loudness = data.attrib.get('v')
+
```

Comment:
Move to the bottom of the file.

```suggestion
@utils.registerPlexObject
class Level(PlexObject):
    """ Represents a single loudness Level sample for an AudioStream.

        Attributes:
            loudness (float): Loudness level value
    """
    def _loadData(self, data):  
        """ Load attribute values from Plex XML response. """
        self.loudness = utils.cast(float, data.attrib.get('v'))
```

Reply:
Moved.

Reply:
You lost all the other changes when you moved it.

Reply:
I don't see this? I still see all your suggested changes in the current version of the files. Please confirm what you're missing and I'll fix asap.